packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -545,22 +550,90 @@ function boundAndSnapToAngle(
 
     const knownSide = magnitude(
         // @ts-expect-error - TS2345 - Argument of type 'number[]' is not assignable to parameter of type 'readonly Coord[]'.
-        vector(coords[rel(-1)], coords[rel(1)]),
+        vector(coordsCopy[rel(-1)], coordsCopy[rel(1)]),
     );
 
     const onLeft =
```

Comment:
Non-blocking nit: I'd ALSO love it if there could be a comment here explaining what the onLeft code is doing / how it affects the offset! 

Since this is shared with both components, I wonder if it could be a little function so the comment doesn't need to be repeated. I'll leave this to your discretion, however, as perhaps this is a small component that simply isn't worth separating out.

Reply:
I would love to pull out some functions that are used in both, but I think that requires a bit more refactoring. And I tried it a bit, but we'd have to pass in a lot of information that makes it a bit harder to parse. The comment should be added now though :)